frontend/src/composables/credential/useCloudProviderBinding.js

```diff
@@ -40,51 +38,65 @@ export const useCloudProviderBinding = (binding, options = {}) => {
   const {
     shootStore = useShootStore(),
     cloudProfileStore = useCloudProfileStore(),
-    gardenerExtensionStore = useGardenerExtensionStore(),
     credentialStore = useCredentialStore(),
   } = options
   const { shootList } = storeToRefs(shootStore)
-  const { cloudProviderBindingList } = storeToRefs(credentialStore)
+  const { infrastructureBindingList } = storeToRefs(credentialStore)
   const { sortedProviderTypeList } = storeToRefs(cloudProfileStore)
-  const { dnsProviderTypes } = storeToRefs(gardenerExtensionStore)
+
+  // Resource
+  const resourceName = computed(() =>
+    get(binding.value, ['metadata', 'name']),
```

Comment:
in the codebase, search for regex `() =>\n`
either write as
```js
  const resourceName = computed(() => get(binding.value, ['metadata', 'name']))
```
or
```js
  const resourceName = computed(() => {
    return get(binding.value, ['metadata', 'name'])
  })
```

Reply:
done